kalite/control_panel/tests/control_panel.py

```diff
@@ -551,60 +557,60 @@ def test_device_log_csv_endpoint(self):
         self.assertEqual(len(rows), 2, "API response incorrect")
 
 
-class CSVExportBrowserTests(CSVExportTestSetup, BrowserActionMixins, CreateAdminMixin, KALiteBrowserTestCase):
+# class CSVExportBrowserTests(CSVExportTestSetup, BrowserActionMixins, CreateAdminMixin, KALiteBrowserTestCase):
```

Comment:
I am happy to take on 2. Creating an issue for these tests with a 0.16 timeline on them should be sufficient for that.